content/payments/upi-deeplinks/refunds.mdx

```diff
@@ -361,11 +340,17 @@ Check status of refunds created against a particular `batchID`.
 
 `Created`: This is when the merchant initiates a refund request via APIs or through The Bridge (Setu dashboard). The refund or its processing has not taken place at this point.
 
-`Marked for Refund`: This status indicates that we have received money from the bank for the underlying transaction, but the refund is yet to be reconciled with the settlement calculation for the day.
+`Initiated`: This is when the refund request has been intimated to the banking partner and begins processing the refund. This is the terminal status of a refund, as NPCI does not share information about when a refund hits the customer’s bank account.
 
-`Queued for Refund`: This status indicates that we are able to adjust the refund amount against the settlement amount.
+`Pending`: This status is implied in case the refunds have not been processed due to a low balance or the merchant not transacting for a while. There are 2 reasons to elaborate on this status:
```

Comment:
For a transaction that happened yesterday with a refund raised shortly after, we would be running our clearing process today. In such a case, the refund will move to an `Pending` state briefly (between the time of clearing initiation and completion), before moving to an `Initiated` state. But the current description in the PR still remains accurate for the notification section. Do you think we should clarify this difference elsewhere? (Wherein we highlight that seeing a `Pending` state in the Refund Report for example is a call for action only if the `Pending Reason` column has one of the aforementioned reasons filled in too).

Reply:
The more I think about it, the reasons listed below are not shared in the notification payload itself. The `reason` key that we send in the notification is the _reason why/how the refund was requested in the first place_, which is particularly useful for Setu-Initiated Refunds. We can perhaps indicate that the reasons mentioned below, at least at the time of writing, are documented here for the merchant's understanding and not a part of the notification payload.